### PR TITLE
Add google-cloud-storage and its dependency google-resumable-media.

### PR DESCRIPTION
**Pull request by johanneskoester**

### CODE REVIEW

**Comment by johanneskoester** [comment]
@conda-forge/staged-recipes: ready for review.

File: recipes/google-cloud-storage/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests >=2.0.0
```
Comment (synapticarbors) [inline recipes/google-cloud-storage/meta.yaml]: It doesn't look like `requests` is required at build time.

File: recipes/google-cloud-storage/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - google.cloud.storage
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-cloud-python
+  summary: Python Client for Google Cloud Storage
+  doc_url: https://googlecloudplatform.github.io/google-cloud-python/stable/google-cloud-api.html
+  dev_url: https://github.com/GoogleCloudPlatform/google-cloud-python/tree/master/storage
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: {{ RECIPE_DIR }}/LICENSE
```
Comment (synapticarbors) [inline recipes/google-cloud-storage/meta.yaml]: Looking on pypi.io, it looks like the `LICENSE` file is packaged with the source, so I don't think you need to add it manually. You can download the source and double check that.

File: recipes/google-resumable-media/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{% set version = "0.0.2" %}
```
Comment (synapticarbors) [inline recipes/google-resumable-media/meta.yaml]: It looks like the newest version on pypi is `0.1.1`. Is there a reason to use the older version?

File: recipes/google-resumable-media/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    - google.resumable_media
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-resumable-media-python
+  summary: Utilities for Google Media Downloads and Resumable Uploads
+  doc_url: https://googlecloudplatform.github.io/google-resumable-media-python
+  dev_url: https://github.com/GoogleCloudPlatform/google-resumable-media-python
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: {{ RECIPE_DIR }}/LICENSE
```
Comment (synapticarbors) [inline recipes/google-resumable-media/meta.yaml]: Could you open an issue or (preferably) a PR on the source repo to add the license file to the `MANIFEST.in` and then xref that issue in a comment in the recipe as a reminder to patch it later.

File: recipes/google-cloud-storage/meta.yaml (hunk 1 above)
Comment (johanneskoester) [inline recipes/google-cloud-storage/meta.yaml]: It is. The build was failing without.

File: recipes/google-cloud-storage/meta.yaml (hunk 2 above)
Comment (johanneskoester) [inline recipes/google-cloud-storage/meta.yaml]: Ok.

File: recipes/google-resumable-media/meta.yaml (hunk 3 above)
Comment (johanneskoester) [inline recipes/google-resumable-media/meta.yaml]: Yes. google-cloud-storage pins to 0.0.2.

File: recipes/google-resumable-media/meta.yaml (hunk 4 above)
Comment (johanneskoester) [inline recipes/google-resumable-media/meta.yaml]: I can open an issue, yes.

File: recipes/google-resumable-media/meta.yaml (hunk 3 above)
Comment (johanneskoester) [inline recipes/google-resumable-media/meta.yaml]: The newest version does not anymore, but this is not yet released.

**Comment by johanneskoester** [comment]
Thanks!

**Pull request merged**